### ISSUE TITLE
Covert Art Image Path

### ISSUE THREAD

**Issue by Keelan** [tidusjar/PlexRequests.Net#716]
If this is a bug report please make sure you have filled the following in:

Seems to be an issue with the cover art on the request page the images are requesting paths such as:

```
<img class="img-responsive" src="https://image.tmdb.org/t/p/w150/https://image.tmdb.org/t/p/w150//Arq0dOHneyM2mQOftr0vPLUE94F.jpg" alt="poster">
```

(If it's not a bug and a feature request then just remove the below)

#### Plex Requests.Net Version:
1.9.7

#### Operating System:
ubuntu 15.10

#### Mono Version:
4.6.1

#### Applicable Logs (from `/logs/` directory or the Admin page):

```
Logs go here (Please make sure you remove any personal information from the logs)
```

#### Reproduction Steps:

View request page

**Comment by Keelan** [comment]
Sorry i did do a search but i was searching cover art. I didn't realize there was an issue open for Posters already. 

https://github.com/tidusjar/PlexRequests.Net/issues/675

**Comment by SuperPotatoMen** [comment]
Thanks for reporting it anyway. It will be fixed in the next Version (1.10.0). You can test it if you like. It is on the dev branch.
